lib/rory/request_parameter_logger.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'parameter_filter'
+
+module Rory
+  class RequestParameterLogger
+
+    def initialize(app, logger=nil, filters=[:password, :tin, :ssn, :social_security_number, :file_attachment])
```

Comment:
Some are pretty specific, but I am going on the assumption that if someone is sending `:ssn` etc they are not going to want that logged.